CHANGELOG.md

```diff
@@ -72,6 +74,12 @@
 
 [issue-2567]: https://github.com/googleapis/google-cloud-cpp/issues/2567
 
+**OTHER CHANGES**:
+* fix(bigtable): change default max mutations per batch from 100k to 1k (#7095)
+* cleanup(bigtable): strictly enforce bigtable limits in mutation batcher (#7036)
+* feat(bigtable): cap outstanding mutation number for mutation batcher (#7028)
+* feat(bigtable): start converting bigtable to use `g::c::Options` (#6949)
```

Comment:
* The `MutationBatcher`'s default setting for max mutations per batch was reduced from 100k to 1k. The new value achieves better throughput and avoids errors from exhausting the server. (#7095)

Reply:
Done.